.gitignore

```diff
@@ -18,3 +18,4 @@ node_modules
 npm-debug.log.*
 
 /local
+package-lock.json
```

Comment:
we may not need to add `package-lock.json` here

Reply:
Regarding the discussions around yarn, package-lock.json is the newer NPM version equivalent for part of what yarn does.

I added this as I wasn't sure you wanted these files in the repo (though best practice suggests we should) - I'll remove this line, but not commit my own package-lock.json files, I'll let you do that yourself when you're ready to.